src/utils/config.ts

```diff
@@ -65,5 +74,74 @@ export const resolveConfigPathByNetworkID = async (networkIdentifier: string): P
 	return configFilePath;
 };
 
-// TODO: Implement with the issue https://github.com/LiskHQ/lisk-migrator/issues/55
-export const migrateUserConfig = async (): Promise<any> => true;
+export const createBackup = async (config: Config): Promise<any> => {
+	const backupPath = `${process.cwd()}/backup`;
+	mkdirSync(backupPath, { recursive: true });
+	writeFileSync(resolve(`${backupPath}/config.json`), JSON.stringify(config, null, '\t'));
+};
+
+export const migrateUserConfig = async (
+	config: Config,
+	liskCorePath: string,
+	tokenID: string,
+): Promise<Record<string, unknown>> => {
```

Comment:
Please consider typecasting the response before returning it.
```suggestion
): Promise<ApplicationConfig> => {
```